code-challenges.test.js

```diff
@@ -48,4 +99,37 @@ const dataArray1 = ["array", "object", "number", "string", "Boolean"]
 const dataArray2 = ["string", "null", "Boolean", "string", "undefined"]
 // Expected output: ["array", "object", "number", "string", "Boolean", "null", "undefined"]
 
+// describe combined_Array
+// it(takes in the values from dataArray1), storing them into a new array
+// it (takes in the values from dataArray2 and compares them to the values already in the new array, removing any duplicated values)
+// Expectede output : ["array", "object", "number", "string", "Boolean", "null", "undefined"]
+
 // b) Create the function that makes the test pass.
+
+// Psuedo-code
+// 1. Create a function that takes in the elements of dataArray1 and loops through those elements to get the initial value of array3
+// 2. Using a nested loop, take in values of dataArray2 and loops through those elements, splicing the non-duplicated elements into array3
+// 3. Console logs the combined arrays of dataArray1 and dataArray2, with the duplicate elements removed
+
+Array.prototype.unique = function() {
+  var a = this.concat();
+  // First loop takes in the values in dataArray 1
+  for(var i=0; i<a.length; ++i) {
+      // Second loop takes in values from dataArray2 and compares them to dataArray1, splicing the new values into the array a
+      for(var j=i+1; j<a.length; ++j) {
+        // if value in dataArray2 is a duplicate of value from dataArray2, remove the duplicate values
+        if(a[i] === a[j])
+              a.splice(j--, 1);
+      }
+  }
+
+  return a;
+};
```

Comment:
This function is taking up a lot real estate. If it works, then it gave the expected output but it is a bit to take in. I am glad that you search for ways to refactor. And it looks like you did find ways to reduce the amount of lines you use on lines 129 and 134. However, neither are following the ES6 function syntax.
Look over functions:
https://github.com/learn-academy-2022-hotel/Syllabus/blob/main/javascript/functions.md

basic structure
```javascript
const functionName = () => {
  return
}
```